File: testprojects/scala_main/pipelines/SCALA_DEP_MGMT_ALL/code/src/main/scala/org/main/scla_dep_mgmt/graph/Filter_4.scala
package org.main.scla_dep_mgmt.graph

import io.prophecy.libs._
import org.main.scla_dep_mgmt.config.ConfigStore._
import org.main.scla_dep_mgmt.udfs.UDFs._
import org.main.scla_dep_mgmt.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Filter_4 {

  def apply(spark: SparkSession, in: DataFrame): DataFrame =
    in.filter(
      !col("c9_udf1_c2")
        .like(Config.c_regex1)
        .and(!col("col2").like(Config.c_regex2))
    )

}

File: testprojects/python_main/pipelines/DONOT_OPEN_PYTHON/code/com/main/pythondepmanagement_1/graph/SubGraph_2/Filter_4.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from com.main.pythondepmanagement_1.config.ConfigStore import *
from com.main.pythondepmanagement_1.udfs.UDFs import *

def Filter_4(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.filter((col("`c- short`") > lit(-1)))
